_posts/2016-01-30-iOS-Junior-Interviews.markdown

```diff
@@ -0,0 +1,300 @@
+---
+layout: post
+title: "Interviewing, applying and getting your first job in iOS"
+date: 2016-01-30 11:09
+author: orta
+categories: [mobile, ios, juniors, interviewing]
+---
+
+When I was at the beginning of my career, my first developer job application was to a design agency who were doing Mac development too. It was pretty nerve-racking to apply for my first programming job, and I came into the interview with no idea what to expect. I had just graduated from university and was at few steps of my career, the term I'm using for this stage of my career is being a junior. It is a time period where I would need mentoring, and supervision in order to grow. A good explanation is in [this StackOverflow](http://programmers.stackexchange.com/questions/14914/whats-the-difference-between-entry-level-jr-sr-developers).
```

Comment:
Maybe change 'was at a few steps of my career, the term I'm using for this stage of my career is being a junior' to 'was at a stage in my career I'd call junior-level'.